docs/routing.md

```diff
@@ -55,6 +55,31 @@ Route('/floating-point/{number:float}', floating_point)
 Route('/uploaded/{rest_of_path:path}', uploaded)
 ```
 
+If you need a different converter that is not defined, you can create your own.
+See below an example on how to create a `datetime` convertor, and how to register it:
+
+```python
+from starlette.convertors import Convertor, register_url_convertor
+
+
+class DateTimeConvertor(Convertor):
+    regex = "[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}(.[0-9]+)?"
+
+    def convert(self, value: str) -> dt.datetime:
+        return dt.datetime.strptime(value, "%Y-%m-%dT%H:%M:%S")
```

Comment:
Do we want to fix `dt` import. I mean it's probably clear but will lead to another `Typo ..` PRs 😄

Reply:
I've simplified it, and added the import statement. Goot catch! :)